harvester_e2e_tests/integrations/test_5_vm_networks_interact.py

```diff
@@ -13,7 +13,8 @@
 pytest_plugins = [
     "harvester_e2e_tests.fixtures.api_client",
     "harvester_e2e_tests.fixtures.networks",
-    "harvester_e2e_tests.fixtures.virtualmachines"
+    "harvester_e2e_tests.fixtures.virtualmachines",
+    "harvester_e2e_tests.fixtures.images"
```

Comment:
I don't get this.
Add a new global fixture but no new test case or local fixtures?

Reply:
I was validating that the OS image update didn't break any other tests and I saw that this test wasn't working. It was missing the the images fixture and was failing since it uses `image_opensuse` which is a part of  `fixtures.images`. I added in the fix to this PR since I was already creating it. 

I can add it in another PR, but I thought it was a small enough change to make here. The test passed with the proper